target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -9,6 +9,10 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath10k/pre-cal-pci-0000:01:00.0.bin")
 	case $board in
+	askey,rt4230w)
+		caldata_extract "art" 0x1000 0x3f1f
+		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary ART 0x6) +1)
```

Comment:
So, the addresses are correct, thanks. I'd be interested whether there are addresses in 0x1002 and 0x5006 as well.
Can you give a table with
0x0
0x6
0x1002
0x5006
?

The last byte of the addresses would be enough.

Further reading: https://openwrt.org/docs/guide-developer/mac.address#find_out_about_flash_locations

Reply:
oh god the last byte of the mac addresses for wireless don't match up. Umm give me a few minutes to fix that.

Reply:
Turns out it was originally getting the ethernet mac addresses and adding 1 or 2 to the last byte. The last byte is the only different one between ethernet and wifi mac addresses so I didn't catch that.